src/services/upload-manager.ts

```diff
@@ -100,7 +176,7 @@ function getErrorDetails(err: any) {
   }
 
   if (err.response?.data?.error) {
-    return JSON.stringify(err.response.data.error);
+    return err.response.data.error.message ?? JSON.stringify(err.response.data.error);
```

Comment:
if there is an error, return the message?  i'm unclear no this flow.

Reply:
I was also not sure why error was being stringified so i assumed it had to be some versions of CHT returning different error objects. In my tests with CHT 4.16.0, the error object had a message field so i check for that before falling back to the previous implementation